client/src/components/DataFiles/DataFilesModals/DataFilesModals.js

```diff
@@ -24,7 +24,7 @@ export default function DataFilesModals() {
       <DataFilesPreviewModal />
       <DataFilesMoveModal />
       <DataFilesCopyModal />
-      <DataFilesUploadModal />
+      <DataFilesUploadModal direction="vertical" density="default" />
```

Comment:
Juan and I chatted. There need only be one property, probably `layout` with options:
- `compact`: show _either_ upload zone _or_ file listing
- `default`: show _both_ upload zone _and_ file listing